locallibrary/catalog/models.py

```diff
@@ -1,3 +1,97 @@
 from django.db import models
+from django.urls import reverse
+import uuid
+from datetime import date
+from django.contrib.auth.models import User
 
-# Create your models here.
+class Genre(models.Model):
+    name = models.CharField(
+        max_length=200,
+        help_text="Enter a book genre (e.g. Science Fiction, French Poetry etc.)"
```

Comment:
toàn bộ các text (vd như nội dung của `help_text`) trong models, views, template thì em translate sử dụng `I18n translation` em nhé
tham khảo: https://docs.djangoproject.com/en/4.1/topics/i18n/translation/